src/card.ts

```diff
@@ -146,4 +146,39 @@ export class CardApi {
     }
     return { success: !resp?.payload?.errorCode, body: { ...resp.payload } }
   }
+
+  /*
+   * Function to take a User token, and a series of search arguments - most of
+   * which are optional- as input, pass them to Marqeta, and have them return
+   * any Marqeta Cards for the User that fit the criteria. If no search
+   * arguments are given, this returns *all* the Card in Marqeta for the User
+   * associated with the token.
+   */
+  async listByUser(token: string, options: {
+    count?: number,
+    startIndex?: number,
+    fields?: string[],
+    sortBy?: string,
+  } | undefined = {}): Promise<{
+    success: boolean,
+    body?: CardList,
+    error?: MarqetaError,
+  }> {
```

Comment:
This signature looks odd... I think it's the same as:

```typescript
   async listByUser(token: string, options?: {
     count?: number,
     startIndex?: number,
     fields?: string[],
     sortBy?: string,
   } = {}): Promise<{
     success: boolean,
     body?: CardList,
     error?: MarqetaError,
   }> {
```

but if the `options` have a default value - I'm not sure that we need the `?`... so it should be sufficient to have:

```typescript
   async listByUser(token: string, options: {
     count?: number,
     startIndex?: number,
     fields?: string[],
     sortBy?: string,
   } = {}): Promise<{
     success: boolean,
     body?: CardList,
     error?: MarqetaError,
   }> {
```